eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -199,6 +201,9 @@ instance Crypto c => EraScript (AlonzoEra c) where
       PlutusScript PlutusV1 _ -> "\x01"
       PlutusScript PlutusV2 _ -> "\x02"
 
+instance Era era => ToJSON (AlonzoScript era) where
+  toJSON = String . serializeAsHexText
```

Comment:
@JaredCorduan This is where the serialization changes made to `cardano-ledger-binary`  come into play.

Reply:
thanks! makes sense now!